api/mesh/v1alpha1/metrics.proto

```diff
@@ -90,3 +93,25 @@ message PrometheusEnvoyConfig {
   // least once). If nil, then it is treated as false.
   google.protobuf.BoolValue usedOnly = 2;
 }
+
+// PrometheusEnvoyConfig defines Tls configuration for Prometheus listener.
+message PrometheusTlsConfig {
+  enum Mode {
+    // control-plane delivers certificates to the prometheus listener.
+    // This should be used when prometheus is running inside the Mesh.
+    activeMTLSBackend = 0;
+    // In this way user is resposible for providing certificates to dataplanes.
+    // Path for the certificte and the key needs to be provided to the dataplane
+    // by environments variables:
+    // * KUMA_DATAPLANE_RUNTIME_METRICS_CERT_PATH
+    // * KUMA_DATAPLANE_RUNTIME_METRICS_KEY_PATH
+    delegated = 1;
```

Comment:
Is this maybe delegatedTLS (the active backend does mtls, this is a just server side TLS).

Reply:
changed to `providedTLS`